themes/default/content/blog/property-search/index.md

```diff
@@ -0,0 +1,269 @@
+---
+
+When we launched [Resource Search for Pulumi Cloud](/blog/pulumi-insights) in April we extended the capability of Pulumi Cloud beyond just managing infrastructure as code to managing your cloud footprint. Today, we're excited to announce that we are taking this a step further and exposing a much larger set of data to search over: resource properties. Pulumi Property Search enables you to search on a wide array of resource properties, everything from instance type, to database version, to region and availability zone, to whether resources are open to the internet.
+
+<!--more-->
+
+Property Search is applicable across all cloud resources, whether they reside in AWS, Google Cloud, Azure, or any of Pulumi's 100+ cloud providers. Pulumi Cloud Resource Search is available on the Enterprise and Business Critical editions of Pulumi Cloud.
+
+### See it in action
+
+![Searching on resource properties in the Pulumi Cloud console](properties.gif)
+
+## Feature Overview
+
+Cloud resources, regardless of the provider, come with various properties that define their behavior, performance, location, and more. With Pulumi Cloud Property Search, you can now search resources based on these properties. Whether you're trying to find resources running in a specific region, instances with a particular tag, or databases running a certain engine version, searching over properties with Pulumi Cloud streamlines and simplifies your tasks.
+
+Using the new feature is simple and straightforward. Go to the Pulumi console, navigate to the search bar, and start typing the property you're interested in. For example, if you're searching for all AWS RDS instances running MySQL with a particular engine version, simply type .engineVersion:5.7 and press enter.
+
+The ability to search for resources based on their properties is a powerful tool for optimizing cloud resource management. It provides deeper insights into your infrastructure and aids in identifying resources that are misconfigured, out-of-date, or not adhering to your organization's best practices. Essentially, it's a significant step towards more proactive, informed, and efficient management of your cloud environment.
+
+Since launching Resource Search, a majority of Pulumi customers have used it, making it the most quickly adopted Pulumi feature. One of these customers is Alkria, a company that reinvents networking for the cloud era with global unified network infrastructure delivered as-a-service. “I’m making developers at Alkira significantly more productive while also making my job easier using Pulumi’s IaC platform and features like Pulumi Insights and Deployments,” said Santosh Dornal, head of software test & DevOps, Alkira. “I can get developers using IaC immediately with Pulumi Deployments and its GitHub integration, while Pulumi Insights makes it really easy to find idle developer environments that need to be shut down, which reduces our cloud costs.”
```

Comment:
```suggestion
Since launching Resource Search, a majority of Pulumi Cloud customers have used it, making it the most quickly adopted feature of Pulumi Cloud. One of these customers is Alkria, a company that reinvents networking for the cloud era with global unified network infrastructure delivered as-a-service. “I’m making developers at Alkira significantly more productive while also making my job easier using Pulumi’s IaC platform and features like Pulumi Insights and Deployments,” said Santosh Dornal, head of software test & DevOps, Alkira. “I can get developers using IaC immediately with Pulumi Deployments and its GitHub integration, while Pulumi Insights makes it really easy to find idle developer environments that need to be shut down, which reduces our cloud costs.”
```

Do we want to move the quote into a quote block to highlight it a bit more explicitly?  Or else perhaps move it into the conclusion if it interrupts the flow?